.github/workflows/dev-container.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow builds a docker image with the Dev Container CLI (https://github.com/devcontainers/cli)
+# 
+name: 'Build Dev Container Image' 
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.devcontainer/**'
+  schedule:
+    # https://crontab.guru/#0_0_*_*_1
+    - cron:  '0 0 * * 1'
+  
+
+permissions:
+  contents: read
+  # Push images to GHCR.
+  packages: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build Base Dev Container Image
+        uses: devcontainers/ci@v0.3
+        with:
+          imageName: ghcr.io/${{ github.repository_owner }}/serenity
```

Comment:
Hm, using `ghcr.io/SerenityOS/serenity` as the Docker image name is maybe too generic if this is specific to devcontainers? Could we maybe call it `serenity-devcontainer` instead?

Reply:
fair enough! Will make that change.